routers/web/org/projects.go

```diff
@@ -678,3 +693,66 @@ func MoveIssues(ctx *context.Context) {
 
 	ctx.JSONOK()
 }
+
+// MoveColumns moves or keeps columns in a project and sorts them inside that project
+func MoveColumns(ctx *context.Context) {
+	if ctx.Doer == nil {
```

Comment:
Why it needs this check? I do not think it could happen

Reply:
d300bfd